src/Azure/Shared/Storage/AzureTableDataManager.cs

```diff
@@ -756,11 +756,6 @@ private async ValueTask<CloudTableClient> GetCloudTableCreationClientAsync()
             try
             {
                 CloudTableClient creationClient = await GetCloudTableClientAsync();
-                if (!string.IsNullOrEmpty(this.StoragePolicyOptions.PreferredRegion))
```

Comment:
yes, we should add this back, so it can read from the nearest region

---
In reply to: [572320655](https://github.com/juyuz-dev/orleans/pull/7#discussion_r572320655) [](ancestors = 572320655)

Reply:
Added it back